ansible/environments/mac/hosts

```diff
@@ -4,25 +4,25 @@
 ansible ansible_connection=local
 
 [edge]
-172.17.0.1          ansible_host=172.17.0.1 ansible_connection=local
+192.168.99.100          ansible_host=192.168.99.100ansible_connection=local
 
 [controllers]
-controller0         ansible_host=172.17.0.1 ansible_connection=local
-controller1         ansible_host=172.17.0.1 ansible_connection=local
+controller0         ansible_host=192.168.99.100ansible_connection=local
+controller1         ansible_host=192.168.99.100ansible_connection=local
 
 [kafka]
-172.17.0.1          ansible_host=172.17.0.1 ansible_connection=local
+192.168.99.100         ansible_host=192.168.99.100ansible_connection=local
 
 [invokers]
-invoker0            ansible_host=172.17.0.1 ansible_connection=local
-invoker1            ansible_host=172.17.0.1 ansible_connection=local
+invoker0            ansible_host=192.168.99.100ansible_connection=local
+invoker1            ansible_host=192.168.99.100ansible_connection=local
 
 ; db group is only used if db_provider is CouchDB
 [db]
-172.17.0.1          ansible_host=172.17.0.1 ansible_connection=local
+192.168.99.100         ansible_host=192.168.99.100ansible_connection=local
 
 [redis]
-172.17.0.1          ansible_host=172.17.0.1 ansible_connection=local
+192.168.99.100         ansible_host=192.168.99.100ansible_connection=local
 
 [apigateway]
-172.17.0.1          ansible_host=172.17.0.1 ansible_connection=local
+192.168.99.100         ansible_host=192.168.99.100ansible_connection=local
```

Comment:
don't change this file, docker-machine is the one using `192.168.99.100`

Reply:
pretty sure this is an accidental commit.

Reply:
Fixed.